pxr/imaging/hdx/colorCorrectionTask.cpp

```diff
@@ -453,9 +454,14 @@ HdxColorCorrectionTask::_CreateOpenColorIOResourcesImpl(
         sampDesc.addressModeU = HgiSamplerAddressModeClampToEdge;
         sampDesc.addressModeV = HgiSamplerAddressModeClampToEdge;
 
+        std::vector<float> lutVector = std::vector<float>(
+            lutValues,
+            lutValues + (valueCount * sizeof(float))
```

Comment:
This bounds calculation isn't correct, but only noticed after we started using the patch on Windows. I'll commit this once we've had a chance to use it a while.

```suggestion
            lutValues + valueCount
```